docs/en/sql-reference/data-types/nested-data-structures/nested.md

```diff
@@ -101,4 +103,8 @@ For a DESCRIBE query, the columns in a nested data structure are listed separate
 
 The ALTER query for elements in a nested data structure has limitations.
 
+## Inserting {#inserting}
+
+To insert `Array(Struct)` values into `Nested` columns, you must switch on the [input_format_parquet_import_nested](../../../operations/settings/settings.md#input_format_parquet_import_nested), [input_format_arrow_import_nested](../../../operations/settings/settings.md#input_format_arrow_import_nested) or [input_format_orc_import_nested](../../../operations/settings/settings.md#input_format_orc_import_nested) settings for [Parquet](../../../interfaces/formats.md#data-format-parquet), [Arrow](../../../interfaces/formats.md#data_types-matching-arrow) and [ORC](../../../interfaces/formats.md#data-format-orc) input formats respectively.
+
```

Comment:
We should remove this at all, the changes are related only to input formats and we shouldn't write about them here.